### PR TITLE
Fix bug on translation

### PR DESCRIPTION
**Pull request by roliverosc**

### CODE REVIEW

File: resources/language/resource.language.es_es/strings.po [hunk 1]
```diff
@@ -1097,7 +1097,7 @@ msgstr "Reproducir con subtítulos"
 
 msgctxt "#30703"
 msgid "Are you sure?"
-msgstr "¿Estás seguro?"
+msgstr "¿Está seguro de que desea cerrar la sesión?"
```
Comment (anxdpanic) [inline resources/language/resource.language.es_es/strings.po]: Can you revert this change, it may be used for more than log out. The dialog title should give the context.

Comment (roliverosc) [reply resources/language/resource.language.es_es/strings.po]: Ok sorry @anxdpanic .

**Comment by anxdpanic** [comment]
no worries at all 👍
